system/crypttab.py

```diff
@@ -74,11 +74,17 @@
 
 EXAMPLES = '''
 - name: Set the options explicitly a device which must already exist
-  crypttab: name=luks-home state=present opts=discard,cipher=aes-cbc-essiv:sha256
+  crypttab:
```

Comment:
I wonder if it's worth adding a comment to remind people to quote `opts` if it contains `:`, what do you think?